src/main/java/org/junit/runner/Result.java

```diff
@@ -137,7 +137,7 @@ public void testIgnored(Description description) throws Exception {
 
         @Override
         public void testAssumptionFailure(Failure failure) {
-            // do nothing: same as passing (for 4.5; may change in 4.6)
+            ignoreCount.getAndIncrement();
```

Comment:
In #1294 (currently targeted for 4.13) we are proposing updating this class to collect assumption failure counts, so I am not sure we should make this change.

Reply:
Oh great! in my opinion add a counter for the assumption failures would be great in order to know exactly how many did the test have